packages/frontend/src/components/DashboardTiles/TileBase/TileBase.styles.tsx

```diff
@@ -42,6 +43,8 @@ export const HeaderContainer = styled.div<HeaderContainerProps>`
                 }
             `
             : ''}
+    ${(props) =>
+        !props.hasTitle ? `margin-top: -5px; margin-bottom: -5px;` : ''}
```

Comment:
do we still need this? not sure what does this fix actually

Reply:
it just makes the header more compact when there's no title